solr/example/films/vectors/README.md

```diff
@@ -23,6 +23,12 @@ We present in this directory the Python scripts that were used to create the `fi
 
 To replicate the example you have to run the `create_model.py` script first, followed by `create_dataset.py`. We will describe and discuss each of these scripts below.
 
+## Setup
+
+```
+pip install sentence-transformers
```

Comment:
```suggestion
pip install -r requirements.txt
```

is better probably. not yet sure though how to best create such a `requirements.txt` file.

Reply:
also, if the list is short, one reason i like the `pip install sentence-transformers` is then you see exactly what is going on, versus having to check some sort of `requirements.txt` file!